version/version.go

```diff
@@ -16,5 +16,5 @@ package version
 
 var (
 	// Version of Kogito Operator
-	Version = "2.0.0-snapshot"
+	Version = "7.11.0"
```

Comment:
we still discussing it, see the Kogito Cloud channel on gchat.

Reply:
Currently I had initialize it with 7.11.0 as suggested by Zanini. In below comment. We could change it once we fialized it in gchat conversation